src/components/Missions/Missions.js

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions, toggleJoinMissions } from '../../redux/missionsReducer';
+import styles from './missions.module.css';
+
+let fetched = false;
+const Missions = () => {
+  const missionsState = useSelector((state) => state.missionsReducer.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!fetched) {
+      dispatch(fetchMissions());
+      fetched = true;
+    }
+  }, [dispatch]);
+
+  const clickHandler = (e) => {
+    const buttonId = e.target.getAttribute('data-target');
+    dispatch(toggleJoinMissions(buttonId));
+  };
+
+  return (
+    <table className={styles.table}>
+      <tbody>
+        <tr>
+          <th>Mission</th>
+          <th>Description</th>
+          <th>Status</th>
+          <th> </th>
+        </tr>
+        {missionsState.map(
+          ({
+            mission_name: missionName,
+            mission_id: missionId,
+            description,
+            reserved,
+          }) => (
+            <tr key={missionId}>
+              <td className={missionName}>{missionName}</td>
+              <td>{description}</td>
+              <td className={styles.badge}>
+                <p
+                  className={
+                    reserved ? styles.missionReserved : styles.notReserved
+                  }
+                >
+                  {reserved ? 'ACTIVE MEMBER' : 'NOT A MEMBER'}
+                </p>
```

Comment:
- [ ] @team kindly check the styling of the active member status, kindly add some space between the two words, you can reduce the font size so that you can achieve this.
![image](https://user-images.githubusercontent.com/61408860/217766277-c602685b-16fc-442b-9c47-15889dc1c50f.png)